tests/providers/amazon/aws/utils/eks_test_constants.py

```diff
@@ -0,0 +1,256 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+"""
+This file should only contain constants used for the EKS tests.
+"""
+import os
+import re
+from enum import Enum
+from typing import Dict, List, Pattern, Tuple
+
+from boto3 import Session
+
+CONN_ID = "eks"
+DEFAULT_MAX_RESULTS = 100
+FROZEN_TIME = "2013-11-27T01:42:00Z"
+PACKAGE_NOT_PRESENT_MSG = "mock_eks package not present"
+PARTITIONS: List[str] = Session().get_available_partitions()
```

Comment:
Is it needed?  Can you use a fixed value here?

Reply:
Removed in upcoming revision.

Reply:
Corrected in https://github.com/apache/airflow/pull/16571/commits/76c7ad635948191764a323a9be03fa965fc91ff5